docs/pipeline.rst

```diff
@@ -32,24 +32,131 @@ Pre-configured Pipelines
-+----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| keyword              | ``["intent_classifier_keyword"]``                                                                                                                                 |
-+----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-| tensorflow_embedding | ``["intent_featurizer_count_vectors", "intent_classifier_tensorflow_embedding"]``                                                                                 |
-+----------------------+-------------------------------------------------------------------------------------------------------------------------------------------------------------------+
-
-Creating your own pipelines is possible by directly passing the names of the components to Rasa NLU in the ``pipeline``
-configuration variable, e.g.
+Here is a list of the **existing templates**:
+
+spacy_sklearn
+~~~~~~~~~~~~~
+
+to use it as a template:
+
+.. code-block:: yaml
+
+    language: "en"
+
+    pipeline: "spacy_sklearn
+
+to use the components and configure them separately:
+
+.. code-block:: yaml
+
+    language: "en"
+
+    pipeline:
+    - name: "nlp_spacy"
+    - name: "tokenizer_spacy"
+    - name: "intent_entity_featurizer_regex"
+    - name: "intent_featurizer_spacy"
+    - name: "ner_crf"
+    - name: "ner_synonyms"
+    - name: "intent_classifier_sklearn"
+
+mitie
+~~~~~
+
+to use it as a template:
+
+.. code-block:: yaml
+
+    language: "en"
+
+    pipeline: "mitie"
+
+to use the components and configure them separately:
+
+.. code-block:: yaml
+
+    language: "en"
+
+    pipeline:
+    - name: "nlp_mitie"
+    - name: "tokenizer_mitie"
+    - name: "ner_synonyms"
+    - name: "intent_entity_featurizer_regex"
+    - name: "intent_classifier_mitie"
+
+mitie_sklearn
+~~~~~~~~~~~~~
+
+to use it as a template:
+
+.. code-block:: yaml
+
+    language: "en"
+
+    pipeline: "mitie_sklearn"
+
+to use the components and configure them separately:
+
+.. code-block:: yaml
+
+    language: "en"
+
+    pipeline:
+    - name: "nlp_mitie"
+    - name: "tokenizer_mitie"
+    - name: "ner_mitie"
+    - name: "ner_synonyms"
+    - name: "intent_entity_featurizer_regex"
+    - name: "intent_featurizer_mitie"
+    - name: "intent_classifier_sklearn"
+
+keyword
+~~~~~~~
+
+to use it as a template:
+
+.. code-block:: yaml
+
+    language: "en"
+
+    pipeline: "keyword"
+
+to use the components and configure them separately:
+
+.. code-block:: yaml
+
+    language: "en"
+
+    pipeline:
+    - name: "intent_classifier_keyword"
+
+
+tensorflow_embedding
+~~~~~~~~~~~~~~~~~~~
+
+to use it as a template:
+
+.. code-block:: yaml
+
+    language: "en"
```

Comment:
adding a note here, that theoretically it supports any language that can be tokenized